src/chainparams.cpp

```diff
@@ -335,12 +331,14 @@ class CTestNetParams : public CChainParams {
             ( 591117, uint256S("0x5f6b93b2c28cedf32467d900369b8be6700f0649388a7dbfd3ebd4a01b1ffad8"))
             ( 658924, uint256S("0xed6c8324d9a77195ee080f225a0fca6346495e08ded99bcda47a8eea5a8a620b"))
             ( 703635, uint256S("0x839fa54617adcd582d53030a37455c14a87a806f6615aa8213f13e196230ff7f"))
+            ( 1000000, uint256S("1fe4d44ea4d1edb031f52f0d7c635db8190dc871a190654c41d2450086b8ef0e"))
+            ( 1202214, uint256S("0xa2179767a87ee4e95944703976fee63578ec04fa3ac2fc1c9c2c83587d096977"))
         };
 
         chainTxData = ChainTxData{
-            // Data as of block 00000000c2872f8f8a8935c8e3c5862be9038c97d4de2cf37ed496991166928a (height 1063660)
-            1440601451, // * UNIX timestamp of last checkpoint block
-            1119061,    // * total number of transactions between genesis and last checkpoint
+            // Data as of block 73864d715723d1b4c7cf41a71aafd648455bba323e0951fde47626c6b19a3c0b (height 1202227)
+            1514565004, // * UNIX timestamp of last checkpoint block
```

Comment:
I don't have testnet synced but chain.so lists 1495653305 for the last checkpoint. Or am I just understanding that comment wrong?

Reply:
Ah crud I've forgotten to actually put the last checkpoint into, y'know, the checkpoints.